stable/weave-cloud/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+name: weave-cloud
+version: latest
```

Comment:
Obviously this doesn't work –`[ERROR] Chart.yaml: version 'latest' is not a valid SemVer`.
What is the best practice for versioning a chart that consists of multiple components?
Is it okay to call it `1.0.0` to start with?